Core/clim-basic/clim-basic.asd

```diff
@@ -1,7 +1,7 @@
 (defsystem #:clim-basic
   :depends-on (#:clim-lisp
                #:spatial-trees
-               (:version "flexichain" "1.5.1")
+               #:flexichain
```

Comment:
I had a problem with that before, newer version of flexichain does work just fine but can't be used as McCLIM dependency (namely some ecl fixes). Moreover version present on Quicklisp is sufficient, so there is no real reason to specify the version (like we do not specify version of opticl and such).

Reply:
> I had a problem with that before, newer version of flexichain does work just fine but can't be used as McCLIM dependency (namely some ecl fixes). 

I don't understand this sentence.

> Moreover version present on Quicklisp is sufficient, so there is no real reason to specify the version (like we do not specify version of opticl and such).

I read `(:version "flexichain" "1.5.1")` as "we would like to document the fact that at least version 1.5.1 of flexichain is required". That seems better than not specifying the version especially since this doesn't prevent the use of newer flexichain versions.

Reply:
you are right, I've confused myself. fixed.